ui/src/components/Overall/Overall.tsx

```diff
@@ -0,0 +1,73 @@
+import "../../style/global.css";
+
+import { MetricHealthStatus } from "../../api/Api";
+import styles from "./Overall.module.css";
+
+// Icon imports
+import SpeedIcon from "@mui/icons-material/Speed";
+import HighlightOffIcon from "@mui/icons-material/HighlightOff";
+import DesktopAccessDisabledIcon from "@mui/icons-material/DesktopAccessDisabled";
+
+interface IOverallProps {
+  overall: MetricHealthStatus;
+}
+
+const Overall: React.FC<IOverallProps> = ({ overall }) => {
+  const data = [
+    {
+      header: "Total Metrics",
+      color: "var(--success)",
+      description:
+        "A visual representation of test execution metrics, providing a comprehensive view of your database performance.",
+      value: overall.total_metrics,
+      icon: <SpeedIcon />,
+    },
+    {
+      header: "Validation Failures",
+      color: "var(--failed)",
+      description:
+        "A breakdown of test failures, empowering you to address issues and enhance database robustness.",
+      value: overall.metric_validation_failed,
+      icon: <HighlightOffIcon />,
+    },
+    {
+      header: "Unchecked Metrics",
+      color: "var(--unchecked)",
+      description:
+        "Unmonitored tests to ensure comprehensive coverage, all in one glance. Stay ahead of potential blind spots.",
+      value: 12,
+      icon: <DesktopAccessDisabledIcon />,
+    },
+  ];
+
+  return (
+    <section className={styles.section}>
+      {data.map((item) => (
+        <div className={styles.card} key={item.header}>
```

Comment:
@driptanil Please create Card component and pass props to manage this part to enhance readability